external-crates/move/crates/move-model-2/src/summary.rs

```diff
@@ -88,42 +88,42 @@ pub struct Function {
     pub return_: Vec<Type>,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
 pub enum Visibility {
     Public,
     Friend,
     Package,
     Private,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct TParam {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     pub name: FromSource<Symbol>,
     pub constraints: AbilitySet,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
 pub struct Parameter {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     name: FromSource<Symbol>,
     type_: Type,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
For my own editor settings, is there a rustfmt flag for this?

Reply:
Did you do this manually then? Or is it some other editor setting?

Reply:
I did it manually... :/

Reply:
![image](https://github.com/user-attachments/assets/61abea26-9cdc-4633-8d56-65a6dd37bfa5)

Reply:
For my own edification, what's the recommended order? Alphabetical?

Reply:
I would argue whatever order pleases you. And @tzakian just wasted his time ;)